### ISSUE TITLE
[BUG] config file should be /usr/local/etc/bpytop.cfg on BSDs

### ISSUE THREAD

**Issue by asomers** [aristocratos/bpytop#282]
**Describe the bug**

bpytop looks for its config file in at /etc/bpytop.cfg .  However, on the BSD operating systems /etc is reserved for stuff that's included in the base system.  Programs installed from ports or by hand should place their config files in /usr/local/etc.

**Info**
 - bpytop version: 1.0.62
 - psutil version: 5.8.0
 - FreeBSD 12.2-RELEASE
 - Terminal used: xfce4-terminal
 - Font used: Monospace Regular 12
 - Python version: 3.7.10

**Comment by aristocratos** [comment]
@asomers 
From README.MD:
```
All options changeable from within UI. Config files stored in "$HOME/.config/bpytop" folder
bpytop.cfg: (auto generated if not found)

"/etc/bpytop.conf" will be used as default seed for config file creation if it exists.
```

`/etc/bpytop.conf` is never created by BpyTop, and should only be admin writable since (if it exists) it's used as a template for new config files  in the users home directory at first start of BpyTop.

**Comment by asomers** [comment]
Yes, I saw that.  But /etc/bpytop.conf is not the correct location for a default config file on BSD.  It should be /usr/local/etc/bpytop.conf instead.

**Comment by aristocratos** [comment]
@asomers 
Well I see no issue in adding a conditional for checking `/usr/local/etc/bpytop.conf` instead if *BSD is detected. Would that be the same for OSX  too then?

**Comment by asomers** [comment]
I don't think so, but I'm not sure.  I don't use OSX.

**Comment by aristocratos** [comment]
Since it's true for BSD and OSX was originally based on BSD wouldn't it make it true for OSX too?

**Comment by asomers** [comment]
Not necessarily.  This isn't some deep fundamental design choice.  It's a much higher level convention.  In fact, the most relevant authority might not be OSX itself, but one of the 3rd party package managers like brew.

**Comment by aristocratos** [comment]
@asomers 
Fixed in version 1.0.64
